app/javascript/components/lease_ups/application_page/preferenceValidationUtils.js

```diff
@@ -0,0 +1,9 @@
+export const VALIDATION_CONFIRMED = 'Confirmed'
```

Comment:
nit. I'd be fine with putting this in the same file as preference rank call just since it doesn't look like we're using them elsewhere? I just get mildly annoyed by having lots of validation files. 

If there are other use cases out there then having this in its own file like this is probably worth it.

Reply:
yep I can move it back. At one point this was being used in two files but now it's not